virttest/qemu_devices/qdevices.py

```diff
@@ -756,11 +756,13 @@ def _convert_blkdev_args(args):
         :rtype: dict
         """
         new_args = dict()
+        keep_original_type = ("detect-zeroes")
         for key, value in six.iteritems(args):
-            if value in ('on', 'yes'):
-                value = True
-            elif value in ('off', 'no'):
-                value = False
+            if key not in keep_original_type:
+                if value in ('on', 'yes'):
+                    value = True
+                elif value in ('off', 'no'):
+                    value = False
```

Comment:
From my understanding, the args `detect-zeroes` work on the protocol node, format node, or both. So we could make changes to the specific node, like the former changes:
https://github.com/avocado-framework/avocado-vt/blob/be859217d8af2f16939ee65d355768ddc748d6c2/virttest/qemu_devices/qdevices.py#L895-L900

Reply:
it looks that detect-zeroes is a valid options for any block driver node(refer to man page)

Reply:
> From my understanding, the args `detect-zeroes` work on the protocol node, format node, or both. So we could make changes to the specific node, like the former changes:
> 
> https://github.com/avocado-framework/avocado-vt/blob/be859217d8af2f16939ee65d355768ddc748d6c2/virttest/qemu_devices/qdevices.py#L895-L900

Hello @YongxueHong , according to the codes,
the class you mentioned s a subclass of `QBlockdevNode`. 
From my opinion, we do NOT need to do the changes in each class. All of the subclass use the function `_convert_blkdev_args` of `QBlockdevNode`.

cc @zhencliu 
Thanks.